tauri/src/endpoints.rs

```diff
@@ -138,6 +139,125 @@ pub(crate) fn handle(webview: &mut Webview<'_>, arg: &str) -> crate::Result<()>
           #[cfg(not(rename_file))]
           allowlist_error(webview, error, "renameFile");
         }
+        AppDir { callback, error } => {
```

Comment:
Maybe it would be easier to change all these command types to a single one with a "type" argument or something

Reply:
Okay, I will make it into a single endpoint and inside `path.rs` I will make all the methods private and only expose one method to handle matching the argument to the related private method.

And I will make another endpoint for `ResolvePath` method since it will receive different arguments.

Reply:
Okay, I will make it into a single endpoint and inside `path.rs` I will make all the methods private and only expose one method to handle matching the argument to the related private method.

And I will make another endpoint for `ResolvePath` method since it will receive different arguments.